package.json

```diff
@@ -99,6 +100,7 @@
     "lint": "eslint lib test bin examples",
     "rebuild-all": "npm rebuild && node-gyp rebuild",
     "repl": "node bin/repl.js",
+    "terminal": "node bin/terminal.js",
```

Comment:
This is useful for development because of: https://github.com/npm/npm/issues/7137